Python/compile.c

```diff
@@ -144,17 +149,26 @@
         (c->c_flags->cf_flags & PyCF_ALLOW_TOP_LEVEL_AWAIT) \
         && (c->u->u_ste->ste_type == ModuleBlock))
 
+struct location {
+    int lineno;
+    int end_lineno;
+    int col_offset;
+    int end_col_offset;
+};
+
+#define LOCATION(LNO, END_LNO, COL, END_COL) \
+    ((const struct location){(LNO), (END_LNO), (COL), (END_COL)})
+
+#define NO_LOCATION {-1, -1, -1, -1}
```

Comment:
Could this be
```c
static struct location NO_LOCATION = {-1, -1, -1, -1};
```
?

Reply:
That might work, it will remove another 6 lines or so!